.cargo/config

```diff
@@ -1,2 +1,5 @@
+[doc.extern-map.registries]
```

Comment:
Tiny nit: Please move this section below `#[alias]`.

Reply:
Done